### DISCUSSION TITLE
warning: upgrading @vitejs/plugin-react-swc to 3.3.1 broke this plugin

### DISCUSSION THREAD

**Discussion by stevebeauge** [aklinker1/vite-plugin-web-extension#113] [General]
If it can help future readers (I loose a few hours on this), upgrading `@vitejs/plugin-react-swc` from `3.2.0` to `3.3.1` broke the execution of the addons.

Popups are not rendered and in the console I get `React refresh runtime was loaded twice. Maybe you forgot the base path?` error.

**Comment by aklinker1** [comment]
I've created an issue for this, #114, and will look into it sometime soon.

**Comment by stevebeauge** [comment]
I didn't create an issue because it feels more like an issue with the `@vitejs/plugin-react-swc` package than this project. Am I wrong ?

**Reply by aklinker1** [reply]
I've had problems with the Vue plugin in the past that were related to this plugin. I need to at least look into it and see if it's my problem or theirs.

**Comment by aklinker1** [comment]
I've checked the basic template `pnpm create vite-plugin-web-extension test-project --template react-ts`, and it is working. See #116, there might be other issues with your current project making it incompatible... I'm looking into fixing these.
